### PR TITLE
Add support for Apollo Guidance Computer extension

### PR DESCRIPTION
**Pull request by chrislgarry**
I accidentallied the previous PR by trying to use the web interface to make changes...

### CODE REVIEW

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -34,6 +34,14 @@ ABAP:
   - .abap
   ace_mode: abap
 
+AGC:
```
Comment (larsbrinkhoff) [inline lib/linguist/languages.yml]: Please change the name here, to "Apollo Guidance Computer".

File: samples/AGC/BURN_BABY_BURN--MASTER_IGNITION_ROUTINE.agc [hunk 2]
```diff
@@ -0,0 +1,1059 @@
+# Copyright:	Public domain.
```
Comment (larsbrinkhoff) [inline samples/AGC/BURN_BABY_BURN--MASTER_IGNITION_ROUTINE.agc]: The samples subdirectory should also be renamed.

**Comment by Alhadis** [comment]
[Grammar's finished](https://github.com/Alhadis/language-agc).

File: lib/linguist/languages.yml [hunk 3]
```diff
@@ -34,6 +34,14 @@ ABAP:
   - .abap
   ace_mode: abap
 
+Apollo Guidance Computer:
```
Comment (pchaigno) [inline lib/linguist/languages.yml]: This entry should be right after the `Apex` entry (entries are sorted alphabetically with upper case characters first).
Sorry for the mess :s

**Comment by pchaigno** [comment]
@chrislgarry Could you add @Alhadis' grammar as described in the [contribution guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md)? I think we're :+1: after that :-)

**Comment by Alhadis** [comment]
Hrm, no colour for the language?

**Comment by chrislgarry** [comment]
@Alhadis NASA blue :)  #0B3D91

**Comment by Alhadis** [comment]
Sigh... could've seen that coming. :S I really think the colour-threshold check should raise a *warning* rather than an error. After all, a machine can't judge colours the way the human eye can.

BTW, if anybody's curious how the grammar looks, [here it is](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-text&grammar_url=&grammar_text=%22name%22%3A+%22Apollo+Guidance+Computer%22%0D%0A%22scopeName%22%3A+%22source.agc%22%0D%0A%22fileTypes%22%3A+%5B%22agc%22%5D%0D%0A%22patterns%22%3A+%5B%22include%22%3A+%22%23main%22%5D%0D%0A%0D%0A%22repository%22%3A%0D%0A%09%0D%0A%09%23+Top-level+patterns%0D%0A%09%22main%22%3A%0D%0A%09%09%22patterns%22%3A+%5B%0D%0A%09%09%09%7B%22include%22%3A+%22%23comment%22%7D%0D%0A%09%09%09%7B%22include%22%3A+%22%23number%22%7D%0D%0A%09%09%09%7B%22include%22%3A+%22%23inclusion%22%7D%0D%0A%09%09%09%7B%22include%22%3A+%22%23identifier%22%7D%0D%0A%09%09%09%7B%22include%22%3A+%22%23opcode%22%7D%0D%0A%09%09%5D%0D%0A%09%0D%0A%09%23+Comments%0D%0A%09%22comment%22%3A%0D%0A%09%09%22patterns%22%3A+%5B%7B%0D%0A%09%09%09%0D%0A%09%09%09%23+Comment-lines%0D%0A%09%09%09%22name%22%3A+%22comment.line.number-sign.agc%22%0D%0A%09%09%09%22begin%22%3A+%22%23%22%0D%0A%09%09%09%22end%22%3A+++%22%24%22%0D%0A%09%09%09%22beginCaptures%22%3A%0D%0A%09%09%09%09%220%22%3A+%22name%22%3A+%22punctuation.definition.comment.agc%22%0D%0A%09%09%09%22patterns%22%3A+%5B%0D%0A%09%09%09%09%22match%22%3A+%22TODO%7CFIXME%7CCHANGED%7CXXX%7CIDEA%7CHACK%7CNOTE%7CREVIEW%7CNB%7CBUG%22%0D%0A%09%09%09%09%22name%22%3A+%22disable-todo%22%0D%0A%09%09%09%5D%0D%0A%09%09%7D%2C%7B%0D%0A%09%09%09%0D%0A%09%09%09%23+Annotations%0D%0A%09%09%09%22name%22%3A+%22comment.annotation.numeric.agc%22%0D%0A%09%09%09%22match%22%3A+%22%5E%5C%5Ct%5B-%2B%5D%5C%5Cd%2B%28%3F%3D%5C%5Ct%29%22%0D%0A%09%09%7D%5D%0D%0A%09%0D%0A%09%0D%0A%09%23+Numeric+literal%0D%0A%09%22number%22%3A%0D%0A%09%09%22name%22%3A+%22constant.numeric.agc%22%0D%0A%09%09%22match%22%3A+%22%5B-%2B%5D%5C%5Cd%2B%28%3F%3A%5C%5C.%5C%5Cd%2B%29%3FD%3F%22%0D%0A%09%0D%0A%09%0D%0A%09%23+Include+directive%0D%0A%09%22inclusion%22%3A%0D%0A%09%09%22name%22%3A+%22meta.preprocessor.include.directive.agc%22%0D%0A%09%09%22begin%22%3A+%22%5E%5C%5C%24%22%0D%0A%09%09%22end%22%3A+%22%28%3F%3D%5C%5Cs%29%22%0D%0A%09%09%22beginCaptures%22%3A%0D%0A%09%09%09%220%22%3A+%22name%22%3A+%22punctuation.definition.directive.agc%22%0D%0A%0D%0A%0D%0A%09%23+Programmar-defined+name%2C+such+as+a+label+or+constant%0D%0A%09%22identifier%22%3A%0D%0A%09%09%22name%22%3A+%22entity.name.identifier.agc%22%0D%0A%09%09%22match%22%3A+%22%5E%28%3F%21%5C%5C%24%29%28%3F%3A%5B%5E%23%5C%5Cs%5D%7B1%2C7%7D%5C%5Ct%7C%5B%5E%23%5C%5Cs%5D%7B8%7D%29%22%0D%0A%0D%0A%09%0D%0A%09%23+Opcode%0D%0A%09%22opcode%22%3A%0D%0A%09%09%22name%22%3A+%22meta.opcode.agc%22%0D%0A%09%09%22begin%22%3A+%22%28%3F%3C%3D%5C%5Ct%29%28%5B%5E%23%5C%5Cs%5D%2B%29%28%3F%3D%5C%5Cs%7C%24%29%22%0D%0A%09%09%22end%22%3A+%22%24%7C%28%3F%3D%23%29%7C%5B%5E%23%5C%5Cs%5D%2B%22%0D%0A%09%09%22beginCaptures%22%3A+%221%22%3A+%22name%22%3A+%22keyword.function.opcode.agc%22%0D%0A%09%09%22endCaptures%22%3A+++%220%22%3A+%22name%22%3A+%22variable.parameter.operand.agc%22%0D%0A&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Fchrislgarry%2FApollo-11%2Fblob%2Fmaster%2FLuminary099%2FBURN_BABY_BURN--MASTER_IGNITION_ROUTINE.s&code=https%3A%2F%2Fgithub.com%2Fchrislgarry%2FApollo-11%2Fblob%2Fmaster%2FLuminary099%2FBURN_BABY_BURN--MASTER_IGNITION_ROUTINE.s).

**Comment by chrislgarry** [comment]
:( okay

The following 1 languages have failing color thresholds. Please modify the hex color.

- Markdown (#083FA1) is too close to ["0B3D91", "083FA1"]

**Comment by chrislgarry** [comment]
@Alhadis looks great!

**Comment by Alhadis** [comment]
Thank you! ^_^

@arfon Why does Markdown even *have* a colour registered, anyway? It's a prose language, so it's not showing up in repository statistics bars.

**Comment by arfon** [comment]
> @arfon Why does Markdown even have a colour registered, anyway? It's a prose language, so it's not showing up in repository statistics bars.

Good point. We can probably remove Markdown's colour safely.

**Comment by Alhadis** [comment]
@chrislgarry You'll need to merge the latest changes into your fork (that is, resolve the conflicts).

If #3113 hasn't been merged by the time you're addressing this, feel free to go ahead and delete Markdown's colour in order to use the NASA blue. =)

**Comment by arfon** [comment]
> @chrislgarry You'll need to merge the latest changes into your fork (that is, resolve the conflicts).

> If #3113 hasn't been merged by the time you're addressing this, feel free to go ahead and delete Markdown's colour in order to use the NASA blue. =)

Because two pull requests wasn't sufficient 😜 this PR is replaced by #3115 which is up to date with `master`

**Pull request closed without merging**